cmd/bundle/run.go

```diff
@@ -33,6 +33,12 @@ func promptRunArgument(ctx context.Context, b *bundle.Bundle) (string, error) {
 		inv[title] = k
 	}
 
+	// Include scripts in the prompt options
```

Comment:
What it looks like:

<img width="308" alt="Screenshot 2025-05-13 at 15 45 40" src="https://github.com/user-attachments/assets/2dd190fb-f116-4a7a-b37c-1d5f91d59e16" />

Reply:
For:
```
scripts:
  foo:
    content: echo hello
```